### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by Nico170902** [XiaMoZhiShi/MorphPlugin#27]
The plugins requires protocolib but it doesnt exist for 1.21.1.
what should I do?

**Comment by MATRIX-feather** [comment]
They have dev builds for 1.21.1. Head to their [CI page](https://ci.dmulloy2.net/job/ProtocolLib) and click 5.3.0, then it should start download automatically

**Comment by Nico170902** [comment]
Thank you.  
but is protocoLIB used to display morphs?

**Comment by MATRIX-feather** [comment]
Yes, or the morphs will only visible by installing the client mod (And displaying other players' morphs on the client using the mod is a bit broken rn)
